src/rucio_client/accountlimits.ts

```diff
@@ -0,0 +1,12 @@
+import { streamData } from '../utils/restApiWrapper'
+
+export function get_account_limits(account: string, rucioToken: string) {
+    const url = '/accounts/' + account + '/limits'
+    return streamData(
+        url,
+        {},
+        {
+            'X-Rucio-Auth-Token': rucioToken,
+        },
+    )
```

Comment:
Any particular reason why we decided to create another client module parallel to the one that already exists?

Reply:
Hey @nimishbongale, this PR is still a WIP, however, @elizabethh777 has moved away from the project. So we use this as a foundation for the Rules pages and finish the pending REST connections and processing of data. ;) 

In this case, this part should be moved to the existing client module. @elizabethh777 wrote this while we were still waiting for the client module ;)